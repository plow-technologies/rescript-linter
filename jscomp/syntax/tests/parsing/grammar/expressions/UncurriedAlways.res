@@uncurried

let foo = (x, y) => x + y

let z = foo(. 3, 4)

let bar = (. x, y) => x + y

let b = bar(3, 4)

let w = 3->foo(4)

let a = 3->foo(. 4)
