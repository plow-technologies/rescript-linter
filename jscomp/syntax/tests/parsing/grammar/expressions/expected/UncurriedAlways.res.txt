[@@@uncurried ]
let foo = ((Function$ (fun x -> fun y -> x + y))[@res.arity 2])
let z = ((foo 3 4)[@res.uapp ])
let bar = ((Function$ (fun x -> fun y -> x + y))[@res.arity 2])
let b = ((bar 3 4)[@res.uapp ])
let w = 3 |.u ((foo 4)[@res.uapp ])
let a = 3 |.u ((foo 4)[@res.uapp ])